Podcast/CardTableViewCell.swift

```diff
@@ -148,11 +151,8 @@ class CardTableViewCell: UITableViewCell {
         episodeNameLabel = UILabel(frame: CGRect.zero)
         dateTimeLabel = UILabel(frame: CGRect.zero)
         descriptionLabel = UILabel(frame: CGRect.zero)
-        recommendedLabel = UILabel(frame: CGRect.zero)
-        playLabel = UILabel(frame: CGRect.zero)
-        
         
-        let labels = [episodeNameLabel, dateTimeLabel, descriptionLabel, recommendedLabel, playLabel]
+        let labels = [episodeNameLabel, dateTimeLabel, descriptionLabel]
```

Comment:
the type of this should not be [UILabel?]
you shouldn't have to force unwrap in the for loop